app/views/locations/_form.html.erb

```diff
@@ -51,15 +51,15 @@
             </div>
             <div class="mb-3">
               <%= f.label :'Start Date' , class: 'form-label' %><br>
-              <%= f.text_field :start_date, type: 'date', value: @location.start_date %>
+              <%= f.text_field :start_date, type: 'date', value: @location.start_date.to_date, data: {'action': 'change->location#excludeDatesList', 'location-target': 'startDate'} %>
```

Comment:
I think `@location.start_date` and `.end_date` are already `Date` objects, so `.to_date` is redundant.